README.md

```diff
@@ -1,116 +1,170 @@
-This is a readme for the RETNET team
+# Are You Using Retentive Networks? (RetNets) 📝
 
-Make sure to have Python=3.11
-When you use a .sh script, run this to grant permission "chmod +x retnet.sh" for the script you want to use
-Then do this "./retnet.sh" to run the script
+This project focuses on the comparison and analysis of the RetNet vs the Transformer architecture, utilizing Microsoft's TorchScale library for implementation. More information can be found in our paper [Are You Using Retentive Networks?](./301R_Retnet_Paper.pdf).
```

Comment:
the RetNet vs the Transformer architecture
This phrasing looks strange.
perhaps "on the comparison and analysis of the RetNet and the Transformer architectures"?

Reply:
Changed to "This project compares and analyzes the RetNet and the Transformer architectures."